### DISCUSSION TITLE
Komme nicht in den Normal Modus

### DISCUSSION THREAD

**Discussion by maikn70** [jomjol/AI-on-the-edge-device#2327] [General]
Ich habe das Setup vom Wasserzähler abgeschlossen aber nach dem Reboot geht er nicht in den normal modus sondern geht immer wieder in den Setup modus habe das schon mehrere male probiert immer wieder das gleiche. Hat da jemand ne iddee was da los sein könnte?

**Comment by caco3** [comment]
Siehe hier: https://github.com/jomjol/AI-on-the-edge-device/discussions/2253#discussion-5008826 resp. https://github.com/jomjol/AI-on-the-edge-device/issues/2256

**Comment by netonnet1984** [comment]
Probiere mal ein Inkognito tab
